packages/engine/paima-sm/src/cde-generic.ts

```diff
@@ -15,14 +15,23 @@ export default async function processDatum(
   const blockHeight = cdeDatum.blockNumber;
   const payload = cdeDatum.payload;
   const prefix = cdeDatum.scheduledPrefix;
+  const shouldIncludeName =
```

Comment:
what's the rationale for not including the name?

Reply:
just that it's not really needed for non-dynamic primitives, since you already known which one is it from the scheduled prefix. 

that said, the join needs to be done anyway since it's a different table, so it doesn't really save anything right now by not being there. I can remove this logic and always add the field if it's simpler that way.

Reply:
yeah I think it's simpler to just always have it. Adds no computational complexity and might be useful for explorers